gridt/src/app/home/home.page.ts

```diff
@@ -35,15 +33,7 @@ export class HomePage implements OnInit, OnDestroy {
   }
 
   private cancelAllSubscriptions() {
-    if (this.swapLeaderSubscription) {
-      this.swapLeaderSubscription.unsubscribe();
-    }
-    if (this.sendSignalSubscription) {
-      this.sendSignalSubscription.unsubscribe();
-    }
-    if (this.getMovementsSubscription) {
-      this.getMovementsSubscription.unsubscribe();
-    }
+    this.subscriptions.map( subscription => subscription.unsubscribe());
```

Comment:
It is indeed a very good idea to keep this in it's own function, even though it's a one-liner, because this way it is much clearer what it is supposed to be doing.

Reply:
The spacing in the map function call is inconsistent, choose either `call( argument );` or `call(argument);`, but not `call( argument);`.